server/src/main/java/com/ncc/neon/services/BetterFileService.java

```diff
@@ -50,17 +47,13 @@
             try {
                 GetResponse response = elasticSearchClient.get(gr, RequestOptions.DEFAULT);
 
-                // Send 404 if file does not exist in database.
                 if (response.getSource() == null) {
-                    sink.error(new ResponseStatusException(HttpStatus.NOT_FOUND, "File not found."));
+                    sink.success(null);
```

Comment:
The file not existing is a "success"? That's surprising.

Reply:
Worked with @chase-farmer for a (hopefully) cleaner approach.